apps/charterafrica/src/lib/ecosystem/index.js

```diff
@@ -12,8 +12,25 @@ import {
 import api from "@/charterafrica/lib/payload";
 import { ECOSYSTEM_GLOBAL } from "@/charterafrica/payload/utils/collections";
 
+function checkConfig(config) {
+  const configTableNames = [
+    "toolTableId",
+    "contributorTableId",
+    "organisationTableId",
+    "partnersTableId",
+    "socialMediaTableId",
+  ];
+  return configTableNames.reduce((acc, current) => {
+    return acc && !!config?.schema?.[current];
+  }, true);
```

Comment:
```suggestion
  const requiredTables = [
    "toolTableId",
    "contributorTableId",
    "organisationTableId",
    "partnersTableId",
    "socialMediaTableId",
  ];
  const isPresent = (tableName) => config?.schema?.[tableName]
  return requiredTables.every(isPresent);
```

All we want is to test some condition & not interested in the actual value => `.every`/`.some` are better than `.reduce`.